IdentityCore/src/controllers/MSIDRequestControlling.h

```diff
@@ -31,6 +31,7 @@ NS_ASSUME_NONNULL_BEGIN
 @protocol MSIDRequestControlling <NSObject>
 
 - (void)acquireToken:(nonnull MSIDRequestCompletionBlock)completionBlock;
+- (void)shouldSkipAcquireTokenBasedOn:(nonnull NSError *)error;
```

Comment:
1. can we add this flag directly  to "acquireToken" method?
2. Nit: Can we rename it to "skipAcquire..." instead of "should..."?"should" sounds like a dynamic condition to me, while here we know for sure that we want to skip.

Reply:
+1, methods with 'should' prefix usually indicate return type as BOOL but this does not seem to be the case here.